services/app/src/components/gameFooter/GameFooter.jsx

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import './gameFooter.css';
 import PropTypes from 'prop-types';
-import { useWebSockets } from '../../hooks/useWebSockets';
+import { ChessBoardContext } from '../../contexts/ChessBoardContext';
 import GameFooterActive from './GameFooterActive';
 import GameFooterInactive from './GameFooterInactive';
 
 const GameFooter = ({ skipCurrentPlayerMove }) => {
-  const { players } = useWebSockets();
+  const { voters } = useContext(ChessBoardContext)
 
   return (
     <div id="game-footer" className='align-c'>
       {
-        players.length < 2 ? <GameFooterInactive /> : <GameFooterActive skipCurrentPlayerMove={skipCurrentPlayerMove} />
+        voters.length < 2 ? <GameFooterInactive /> : <GameFooterActive skipCurrentPlayerMove={skipCurrentPlayerMove} />
```

Comment:
Could we use `isGameInProgress` here?

Reply:
What if the game is complete?

Reply:
What changes then?

Reply:
Got it, makes sense.

To make it easier to reason about, I'd suggest to introduce either a new variable next to `isGameInProgress` and name it something like this: `isWaitingForGameStart` or any other alternative, that would indicate, that game has not yet started.

Another alternative would be to introduce a new state variable in chessboard context. It would hold a variable for the current state of the game. It will be transformed into enum, after we migrate to TS
```
const GAME_NOT_STARTED = 'GAME_NOT_STARTED';
const GAME_IN_PROGRESS = 'GAME_IN_PROGRESS';
const GAME_FINISHED = 'GAME_FINISHED';

<...>
const [gameState, setGameState] = useState(GAME_NOT_STARTED);
<...>
```

Then we could rewrite this to
```
  const { gameState } = useContext(ChessBoardContext);
  <...>
  return (
    <...>
    {gameState === GAME_NOT_STARTED ? <GameFooterInactive /> : <GameFooterActive />}
  );
```
Much more readable.